step-issuer/templates/service.yaml

```diff
@@ -9,10 +9,12 @@ metadata:
     prometheus.io/scrape: "{{ .Values.service.scrape }}"
   labels:
     control-plane: {{ .Values.service.controlPlane }}
+    {{- include "step-issuer.labels" . | nindent 4 }}
 spec:
   ports:
   - name: {{ .Values.service.targetPorts }}
     port: {{ .Values.service.port }}
     targetPort: {{ .Values.service.targetPorts }}
   selector:
+    {{- include "prometheus-blackbox-exporter.selectorLabels" . | nindent 4 }}
```

Comment:
Because `prometheus-blackbox-exporter.selectorLabels` would be created on a helper of `prometheus-blackbox-exporter` chart, I think for this chart makes more sense to use a new `.Values.service.selectorLabels`.

We should also add here the default:
```tpl
    {{- include "step-issuer.selectorLabels" . | nindent 4 }}
```

Reply:
Hi, 
the prometheus-selector label was an overzealous copy/paste.. Are there use-cases for using different selector labels than control-plane + boilerplate app.kubernetes.io/name and /instance-labels? Or are you suggesting not using the helper-function?

Reply:
I was suggesting adding the helper from step-issuer, what you did in [37f7fbb](https://github.com/smallstep/helm-charts/pull/99/commits/37f7fbb73597333a89d94544a386119684a22254), but I thought you needed more annotations, that's why I suggested the new properties in the values.yaml.

I'll merge the chart as it is right now.